hera_qm/xrfi.py

```diff
@@ -425,65 +420,23 @@ def flag_autos(data, flag_method="detrend_diff_filt", nsig=6, int_count=None, an
         if ant1 == ant2:
             # If antenna is labeled "bad", skip and set weights to zeros
             if antenna_class is not None and antenna_class.is_bad(ant1):
-                rfi_wgts[ant1] = np.zeros_like(v)
+                rfi_wgts[ant1] = np.ones_like(v, dtype=bool)
                 continue
 
             # Estimate the noise standard deviation from data
             noise = np.abs(v) / np.sqrt(int_count / 2)
 
             # Run flagging algorithm
-            rfi_wgts[ant1] = alg_func(v, noise=noise, nsig=nsig, wgts=wgts.get(ant1, np.ones_like(v)), **flag_kwargs)
-            array_weights.append((1 - rfi_wgts[ant1]))
+            rfi_wgts[ant1] = alg_func(
+                v, noise=noise, nsig=nsig, flags=flags.get(ant1, np.zeros_like(v, dtype=bool)), **flag_kwargs
+            )
+            array_weights.append(rfi_wgts[ant1])
 
     # Combine into array-wide weights
     array_weights = np.mean(array_weights, axis=0)
-    array_weights = np.where(array_weights < array_wgts_threshold, 1, 0)
+    array_weights = np.where(array_weights < flag_threshold, False, True)
```

Comment:
These aren't really weights anymore, no? Probably the docstring should change too.

Reply:
Changed the names of these